packages/lesswrong/server/resolvers/anthropicResolvers.ts

```diff
@@ -119,20 +121,49 @@ async function getQueryContextDecision(args: BasePromptArgs): Promise<RagContext
   return parsedResponse.strategy_choice;
 }
 
+async function getProvidedPosts(query: string, context: ResolverContext): Promise<PostsPage[]> {
+  const postIdRegex = /\/([a-zA-Z0-9]{17})(?=[/#?&)]|$)/g;
```

Comment:
This is simply not reliable but I can't think of an amazing solution in 5 seconds.  Something involving our list of routes, probably.

Reply:
Yeah, not reliable. Breaks on pages like /faq, /about
I think it's okay at the level of polish this feature is at.